### PR TITLE
Added code support for X Rebirth

### PR DESCRIPTION
**Pull request by leahcimNosliw**
- Added image for X-Rebirth to images
- Added code support for X-Rebirth
- Added X-Rebirth to the counter in manual

- This is just a first version of the generator for this game, I hope that isn't a problem. I do plan to add more to it over the next couple of weeks.
- Also: great idea!

### CODE REVIEW

File: config/initializers/constants.rb [hunk 1]
```diff
@@ -6025,5 +6025,232 @@ def new_vegas_specials
                 ]
             }
         }
-    }
+    },
+	xrebirth: {
+		title: X-Rebirth
+		background:
```
Comment (Lockyy) [inline config/initializers/constants.rb]: Unfortunately the site won't load with this as it currently is, the title will need to be a string or symbol and we'll need to specify the name of the background image

File: config/initializers/constants.rb [hunk 2]
```diff
@@ -6025,5 +6025,232 @@ def new_vegas_specials
...
+					:'Only use missiles; no other weapons',
+					:'You may only stock 20 missiles at one time',
+					:'You may only stock 10 missiles at one time',
+					:'Do not upgrade beyond Mk1 weapons',
+					:'Do not upgrade beyond Mk1 shields',
+					:'Do not upgrade beyond Mk1 engines',
+					:'Do not buy any ships',
+					:'Trade only with inventory items',
+					:'Dead is dead - if you die; game over',
+					:primary_weapon: {
```
Comment (Lockyy) [inline config/initializers/constants.rb]: :primary_weapon: needs to be replaced with primary_weapon:

File: config/initializers/constants.rb [hunk 3]
```diff
@@ -6025,5 +6025,232 @@ def new_vegas_specials
...
+					:'Fully upgrade 50 stations',
+					:'Build at least one station for any ware at every tech level',
+					:'Build at least one of every possible station',
+					:'Build 25 trading ships',
+					:'Build 50 trading ships',
+					:'Build 100 trading ships',
+					:'Build 25 capital ships',
+					:'Build 50 capital ships',
+					:'Build 100 capital ships',
+					:specific_zone_build: {
```
Comment (Lockyy) [inline config/initializers/constants.rb]: Same issue as primary_weapon above

**Comment by Lockyy** [comment]
Just added a few notes of things to fix before it can be merged. Will also need to reduce the filesize on the background to ~500kb. Gotta think about page load times.

Other than that this look really good and I look forward to adding this to the site.

**Comment by leahcimNosliw** [comment]
I am going to close this pull request and send you an updated one then. Incoming :)

**Pull request closed without merging**